src/charm.py

```diff
@@ -299,7 +299,7 @@ def _reconcile(self):
             component=Component.receiver,
             name="filelog/var-log",
             config=_filelog_receiver_config(
-                include=["/var/log/**"],
```

Comment:
I've said this already, but I'll repeat: do we really need a separate job for `syslog`? Labels and everything are the same, we should probably try `include=["/var/log/**/*.log", "/var/log/syslog"]`